scripts/configuration_helper.lua

```diff
@@ -86,820 +72,386 @@ function sgct.config.single(arg) end
 -- sgct.config.fisheye(msaa=1) -> 1280x720 resolution without multisampling
 function sgct.config.fisheye(arg) end
 
-function sgct.config.cube(arg) end
-
 -- Global variable storing the name of the config function called at initialization
 sgctconfiginitializeString = ""
 
+
+
+
 --[[
 ########################################################################################## 
                             Internal helper functions
 ########################################################################################## 
 ]]--
 
 function generateSingleViewportFOV(down, up, left, right, tracked)
-    return
-[[
-<Viewport ]]..tracked..[[>
-    <Pos x="0.0" y="0.0" />
-    <Size x="1.0" y="1.0" />
-    <PlanarProjection>
-        <FOV down="]]..down..[[" left="]]..left..[[" right="]]..right..[[" up="]]..up..[[" />
-        <Orientation heading="0.0" pitch="0.0" roll="0.0" />
-    </PlanarProjection>
-</Viewport>
-]]
+  local result = {}
+
+  table.insert(result,   [[            {]])
```

Comment:
Good point.  I think at this point, this is the only place where we are doing it from Lua.  Mainly because the configurtion_helper is loaded before we initialize the OpenSpaceEngine which would be the place that would provide a lot of those libraries